framework/Models/PostProcessor.py

```diff
@@ -207,9 +210,9 @@ def submit(self,myInput,samplerType,jobHandler,**kwargs):
         @ Out, None
     """
     kwargs['forceThreads'] = True
-    Model.submit(self,myInput, samplerType, jobHandler,**kwargs)
+    super().submit(myInput, samplerType, jobHandler,**kwargs)
```

Comment:
as above

Reply:
See above comment. In addition, 'samplerType' is needed, since we are calling the method from Model class.

Reply:
See comment below. The samplerType should be encpasulated in the kwargs so usable by only the "entities" that need it.